README.md

```diff
@@ -26,11 +26,11 @@ Adapter, please open an issue here or email
 **Note:** Any supported Runtime SDK can be paired with any Adapter from the same
 language.
 
-| Language   | Runtime SDKs                             | Adapters                                                                                                                                  |
-| ---------- | ---------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
-| Rust       | [Wasmtime](/rust)                        | [Datadog](/rust/src/adapter/datadog.rs), [OpenTelemetry (STDOUT)](/rust/src/adapter/otelstdout.rs), [Zipkin](/rust/src/adapter/zipkin.rs) |
-| Go         | [Wazero](/go)                            | [Datadog](/go/adapter/datadog/), [OpenTelemetry (STDOUT)](/go/adapter/otel_stdout/)                                                       |
-| JavaScript | [Native](/js) (Browser, Node, Deno, Bun) | [Datadog](/js/packages/observe-sdk-datadog)                                                                                               |
+| Language   | Runtime SDKs                             | Adapters                                                                                                                                                                                                                            |
+| ---------- | ---------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
my editor adds these superfluous hyphens.. pretty sure the spec just asks for a single hyphen! not harming anything to leave though I suppose.